tfgrid3/mastodon/Dockerfile

```diff
@@ -1,42 +1,59 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates curl gnupg lsb-release && \
+RUN apt-get update && apt-get install -y \
+    curl \
+    gnupg \
+    lsb-release \
+    wget \
+    apt-transport-https \
+    ca-certificates \
+    ufw \
+    vim \
+    openssh-server \
+    software-properties-common \
+    lsb-release && \
+    # Docker
     curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg && \
     echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
-    $(lsb_release -cs) stable" |  tee /etc/apt/sources.list.d/docker.list > /dev/null && \
-    apt-get update && \
-    DEBIAN_FRONTEND=noninteractive TZ=Europe/Belgium apt-get install -y docker-ce docker-ce-cli containerd.io \
-    openssh-server sudo ufw nano vim && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
-    chmod +x /usr/local/bin/docker-compose
-
-RUN wget -O /sbin/zinit https://github.com/threefoldtech/zinit/releases/download/v0.2.5/zinit && \
-  chmod +x /sbin/zinit 
-
-RUN apt-get update && \ 
-    apt-get install -y iproute2 htop net-tools rsync mc mcedit
-
-RUN apt install -y software-properties-common &&\
-    wget -q http://www.webmin.com/jcameron-key.asc -O- | sudo apt-key add - &&\
-    add-apt-repository "deb [arch=amd64] http://download.webmin.com/download/repository sarge contrib" &&\
-    apt install -y webmin
-
-RUN wget https://github.com/cdr/code-server/releases/download/v3.3.1/code-server-3.3.1-linux-amd64.tar.gz &&\
+    $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+    # Docker-compose
+    curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
```

Comment:
Docker-compose v2 is now available, stable, and should be compatible with docker-compose. the only difference is that it no longer calls `docker-compose` but `docker compose`. if no specific reason to use docker-compose v1, I would suggest using the latest version.

Reply:
used `docker-compose-plugin` instead of `docker-compose`.